.gitmodules

```diff
@@ -25,3 +19,12 @@
 [submodule "contracts/ckb-smt/c/deps/molecule"]
 	path = contracts/ckb-smt/c/deps/molecule
 	url = https://github.com/nervosnetwork/molecule
+[submodule "common/c/deps/ckb-production-scripts"]
+	path = common/c/deps/ckb-production-scripts
+	url = https://github.com/nervosnetwork/ckb-production-scripts.git
+[submodule "common/c/deps/ckb-c-stdlib"]
+	path = common/c/deps/ckb-c-stdlib
+	url = https://github.com/nervosnetwork/ckb-c-stdlib
+[submodule "common/c/deps/molecule"]
```

Comment:
why add molecule submodule

Reply:
it is needed by newly added secp256k1_util